aya/src/maps/mod.rs

```diff
@@ -235,36 +241,44 @@ fn maybe_warn_rlimit() {
 /// eBPF map types.
 #[derive(Debug)]
 pub enum Map {
-    /// A [`Array`] map
+    /// An [`Array`] map.
     Array(MapData),
-    /// A [`PerCpuArray`] map
+    /// A [`PerCpuArray`] map.
     PerCpuArray(MapData),
-    /// A [`ProgramArray`] map
+    /// A [`ProgramArray`] map.
     ProgramArray(MapData),
-    /// A [`HashMap`] map
+    /// A [`HashMap`] map.
     HashMap(MapData),
-    /// A [`PerCpuHashMap`] map
+    /// A [`PerCpuHashMap`] map.
     PerCpuHashMap(MapData),
     /// A [`HashMap`] map that uses a LRU eviction policy.
     LruHashMap(MapData),
     /// A [`PerCpuHashMap`] map that uses a LRU eviction policy.
     PerCpuLruHashMap(MapData),
-    /// A [`PerfEventArray`] map
+    /// A [`PerfEventArray`] map.
     PerfEventArray(MapData),
-    /// A [`SockMap`] map
+    /// A [`SockMap`] map.
     SockMap(MapData),
-    /// A [`SockHash`] map
+    /// A [`SockHash`] map.
     SockHash(MapData),
-    /// A [`BloomFilter`] map
+    /// A [`BloomFilter`] map.
     BloomFilter(MapData),
-    /// A [`LpmTrie`] map
+    /// A [`LpmTrie`] map.
     LpmTrie(MapData),
-    /// A [`Stack`] map
+    /// A [`Stack`] map.
     Stack(MapData),
-    /// A [`StackTraceMap`] map
+    /// A [`StackTraceMap`] map.
     StackTraceMap(MapData),
-    /// A [`Queue`] map
+    /// A [`Queue`] map.
     Queue(MapData),
+    /// A [`CpuMap`] map.
+    CpuMap(MapData),
+    /// A [`DevMap`] map.
+    DevMap(MapData),
+    /// A [`DevMapHash`] map.
+    DevMapHash(MapData),
+    /// A [`XskMap`] map.
+    XskMap(MapData),
     /// An unsupported map type
```

Comment:
LOL you changed it to all variants except the last one!? 💀

Reply:
that or it got lost in a rebase 🤣 
fixed